lib/Dialect/Lattigo/IR/LattigoRLWEOps.td

```diff
@@ -31,6 +31,43 @@ def Lattigo_RLWEGenKeyPairOp : Lattigo_RLWEOp<"gen_key_pair"> {
   );
 }
 
+def Lattigo_RLWEGenRelinearizationKeyOp : Lattigo_RLWEOp<"gen_relinearization_key"> {
+  let summary = "Generates a new RLWE relinearization key";
+  let description = [{
+    This operation generates a new RLWE relinearization key
+  }];
+  let arguments = (ins
+    Lattigo_RLWEKeyGenerator:$keyGenerator,
+    Lattigo_RLWESecretKey:$secretKey
+  );
+  let results = (outs Lattigo_RLWERelinearizationKey:$relinearizationKey);
+}
+
+def Lattigo_RLWEGenGaloisKeyOp : Lattigo_RLWEOp<"gen_galois_key"> {
+  let summary = "Generates a new RLWE Galois key";
+  let description = [{
+    This operation generates a new RLWE Galois key
+  }];
+  let arguments = (ins
+    Lattigo_RLWEKeyGenerator:$keyGenerator,
+    Lattigo_RLWESecretKey:$secretKey,
+    Builtin_IntegerAttr:$galoisElement
```

Comment:
The semantics of this galoisElement should be documented. Is it the `i` in `x -> x^i`?

Reply:
yes. Will add a comment